### PR TITLE
Add additional params for object storage configuration in CreatePipelineServer, fixing tests on disconnected

### PR DESCRIPTION
**Pull request by jgarciao**
- Allows to configure bucket name, region and host. 
- This fixes error when running dsp tests on disconnected using this keyword
- Splits kfp tests in Smoke
- Adds AutomationBugOnDisconnected and ProductBugOnDisconnected to failing tests

### CODE REVIEW

**Review by CFSNM — APPROVED** [review]

File: ods_ci/libs/DataSciencePipelinesKfp.py [hunk 1]
```diff
@@ -126,7 +126,7 @@ def create_run_from_pipeline_func(
             pipeline_params["openshift_server"] = self.api.get_openshift_server()
         if "openshift_token" in pipeline_params:
             pipeline_params["openshift_token"] = self.api.get_openshift_token()
-        print(f"pipeline_params modified with dynamic values: {pipeline_params}")
+        print("pipeline_params modified with dynamic values")
```
Comment (diegolovison) [inline ods_ci/libs/DataSciencePipelinesKfp.py:129]: Why this change?

File: ods_ci/tests/Resources/CLI/DataSciencePipelines/DataSciencePipelinesBackend.resource [hunk 2]
```diff
@@ -11,36 +12,54 @@ ${DSPA_PATH}=      tests/Resources/Files/pipeline-samples/v2/dspa
...
 #    ...    ${database_username}=${EMPTY}    ${database_password}=${EMPTY}
 #    ...    ${database_db_name}=${EMPTY}
     ...    ${dsp_version}=v2
+    ...    ${configure_pip_index}=${TRUE}
 
     Create Secret With Pipelines Object Storage Information    namespace=${namespace}
     ...    object_storage_access_key=${object_storage_access_key}
     ...    object_storage_secret_key=${object_storage_secret_key}
 
+    ${object_storage_endpoint}=    Remove String    ${object_storage_endpoint}    https://    http://
```
Comment (diegolovison) [inline ods_ci/tests/Resources/CLI/DataSciencePipelines/DataSciencePipelinesBackend.resource:34]: Why do you need to remove http or https?

File: ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot [hunk 3]
```diff
@@ -67,11 +77,10 @@ Verify Upload Download In Data Science Pipelines Using The kfp Python Package
     ...    pipeline_params=${upload_download_dict}
     [Teardown]    Projects.Delete Project Via CLI By Display Name    ${PROJECT_NAME}
 
-
 Verify Ods Users Can Create And Run A Data Science Pipeline With Ray Using The kfp Python Package
     [Documentation]    Creates, runs pipelines with regular user. Double check the pipeline result and clean
     ...    the pipeline resources.
-    [Tags]      Tier1
+    [Tags]      Tier1    ExcludeOnDisconnected
```
Comment (diegolovison) [inline ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot:83]: Why Excluded from disconnected?

File: ods_ci/libs/DataSciencePipelinesKfp.py (hunk 1 above)
Comment (jgarciao) [reply ods_ci/libs/DataSciencePipelinesKfp.py:129]: The logs were printing the object storage credentials

File: ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot (hunk 3 above)
Comment (jgarciao) [reply ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot:83]: This test was failing on disconnected

File: ods_ci/libs/DataSciencePipelinesKfp.py (hunk 1 above)
Comment (diegolovison) [reply ods_ci/libs/DataSciencePipelinesKfp.py:129]: Ok. It was introduced for troubleshooting purpose. Replace by `sorted(pipeline_params.keys())`

File: ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot (hunk 3 above)
Comment (diegolovison) [reply ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot:83]: Is it an automation issue or is it an issue with ray and an airgap environment ?

File: ods_ci/libs/DataSciencePipelinesKfp.py (hunk 1 above)
Comment (jgarciao) [reply ods_ci/libs/DataSciencePipelinesKfp.py]: Done

File: ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot (hunk 3 above)
Comment (jgarciao) [reply ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot:83]: I believe it's an automation bug. The image used for the test should be mirrored. I'll write you about this on slack

Comment (apodhrad) [reply ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot:83]: Please do not exclude it. Rather let's fix it ;-)

Comment (jgarciao) [reply ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot]: I've added it back and I'll create a Jira about fixing it when possible

File: ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot [hunk 4]
```diff
@@ -67,11 +77,11 @@
     ...    pipeline_params=${upload_download_dict}
     [Teardown]    Projects.Delete Project Via CLI By Display Name    ${PROJECT_NAME}
 
-
 Verify Ods Users Can Create And Run A Data Science Pipeline With Ray Using The kfp Python Package
     [Documentation]    Creates, runs pipelines with regular user. Double check the pipeline result and clean
     ...    the pipeline resources.
     [Tags]      Tier1
+
```
[the remark this answers is not in the document]
Comment (apodhrad) [reply ods_ci/tests/Tests/1100__data_science_pipelines/1100__data-science-pipelines-kfp.robot:84]: Hi @jgarciao let's reduce the number of such warnings ;-)

**Review by apodhrad — APPROVED** [review]

**Review by diegolovison — APPROVED** [review]

**Review by CFSNM — APPROVED** [review]

File: ods_ci/tests/Resources/Files/pipeline-samples/v2/dspa/dspa-template.yaml [hunk 5]
```diff
@@ -29,7 +29,7 @@ parameters:
   value: "v2"
   name: DSP_VERSION
 - description: Object Storage Bucket Name
-  value: "ods-ci-ds-pipelines"
+  value: "test-bucket"
```
Comment (apodhrad) [inline ods_ci/tests/Resources/Files/pipeline-samples/v2/dspa/dspa-template.yaml:32]: Hi @jgarciao why did you change this bucket name?

Comment (jgarciao) [reply ods_ci/tests/Resources/Files/pipeline-samples/v2/dspa/dspa-template.yaml:32]: The real value is set in keyword `CreatePipelineServer`, this is just an example

**Review by apodhrad — APPROVED** [review]
